src/app.js

```diff
@@ -1 +1,44 @@
 'use strict';
+/* eslint-disable no-console */
+
+const readline = require('readline');
+const path = require('path');
+const { copyFile } = require('./copyFile');
```

Comment:
The imported 'copyFile' function from './copyFile' is not used asynchronously. Consider using async/await or callbacks to handle any errors and ensure proper copying.